pkg/broker/util.go

```diff
@@ -219,7 +223,14 @@ func parametersToSchema(plan apb.Plan) Schema {
 					Required:   createRequired,
 				},
 			},
-			Update: map[string]*schema.Schema{},
+			Update: map[string]*schema.Schema{
+				"parameters": {
+					SchemaRef:  schema.SchemaURL,
+					Type:       []schema.PrimitiveType{schema.ObjectType},
+					Properties: updatableProperties,
+					Required:   updatableRequired,
```

Comment:
Will this always be the set of required parameters that **also** are update-able?